src/test/java/apoc/math/RegressionTest.java

```diff
@@ -0,0 +1,90 @@
+package apoc.math;
+
+import org.apache.commons.math3.stat.regression.SimpleRegression;
+import org.junit.Rule;
+import org.junit.Test;
+import org.neo4j.driver.v1.*;
+import org.neo4j.harness.junit.Neo4jRule;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author <a href="mailto:ali.arslan@rwth-aachen.de">AliArslan</a>
+ */
+public class RegressionTest {
```

Comment:
A test not spinning up a neo4j server but, running it against an embedded instance of Neo4j.  See: https://github.com/neo4j-contrib/neo4j-apoc-procedures/blob/3.1/src/test/java/apoc/coll/CollTest.java